.github/workflows/frontend-deploy.yml

```
@@ -0,0 +1,58 @@
+          node-version: "22"
+
+      - name: Install dependencies
+        run: npm install
+
+      - name: Build project
+        run: npm run build
+        env:
+          VITE_GOOGLE_TAG: ${{ secrets.VITE_GOOGLE_TAG }}
+
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1
```

actionlint (1.7.7)

the runner of "aws-actions/configure-aws-credentials@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)